packages/ibm-products/src/components/ConditionBuilder/ConditionBuilderContent/ConditionBuilderContent.js

```diff
@@ -0,0 +1,83 @@
+import React, { useContext, useEffect, useRef, useState } from 'react';
+import { pkg } from '../../../settings';
+import { Button } from '@carbon/react';
+import { Add } from '@carbon/react/icons';
+import {
+  ConditionBuilderContext,
+  emptyState,
+} from '../ConditionBuilderContext/DataTreeContext';
+
+const blockClass = `${pkg.prefix}--condition-builder`;
+
+function ConditionBuilderContent() {
+  const { rootState, setRootState } = useContext(ConditionBuilderContext);
+  const [isConditionbuilderActive, setIsConditionbuilderActive] =
+    useState(true);
+  const conditionBuilderRef = useRef();
+
+  useEffect(() => {
+    if (!rootState?.groups || rootState?.groups.length == 0) {
```

Comment:
i think you can simplify this to `rootState?.groups?.length === 0`

Reply:
modified as advised